core/src/test/java/io/cucumber/core/plugin/DefaultSummaryPrinterTest.java

```diff
@@ -22,16 +24,25 @@
 class DefaultSummaryPrinterTest {
 
     private final ByteArrayOutputStream out = new ByteArrayOutputStream();
-    private final DefaultSummaryPrinter summaryPrinter = new DefaultSummaryPrinter(out);
+    private DefaultSummaryPrinter summaryPrinter;
     private final EventBus bus = new TimeServiceEventBus(
         Clock.fixed(ofEpochSecond(0), ZoneId.of("UTC")),
         UUID::randomUUID);
+    private Locale originalLocale;
 
     @BeforeEach
     void setup() {
+        originalLocale = Locale.getDefault();
+        Locale.setDefault(Locale.ENGLISH);
+        summaryPrinter = new DefaultSummaryPrinter(out);
         summaryPrinter.setEventPublisher(bus);
     }
 
+    @AfterEach
+    void teardown() {
+        Locale.setDefault(originalLocale);
```

Comment:
This may belong in a seperate PR.

Reply:
Thought it was small enough to be kept here, should I extract it?

Reply:
Please do. I'd like to know why this is needed.

Reply:
https://github.com/cucumber/cucumber-jvm/pull/2010

Reply:
I've fixed #2010. Can you remove these changes?